kappnav.yaml

```diff
@@ -90,7 +90,7 @@ spec:
       containers:
         - name: kappnav-operator
           # Replace this with the built image name
-          image: kappnav/operator:KAPPNAV_VERSION
```

Comment:
I'm guessing this was changed inadvertently. This needs to stay as "KAPPNAV_VERSION". The build is responsible for replacing this value with the correct tag for the kAppNav release. Please fix.

Reply:
I think that the version is automatically replaced with 0.6.0 when operator is built.